Array Sorter/js/arrayHandler.js

```diff
@@ -0,0 +1,91 @@
+class ArraySorter{
+
+
+    swap(arr, indexA, indexB){
```

Comment:
the function is mutating outside variable which is good for imperative programming but I would suggest to read a bit about object oriented programming and functional programming.